spec/adapters/mysql_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Polo::Adapters::MySQL do
+  include Polo::Adapters::MySQL
+
+  let(:netto) do
+    AR::Chef.where(name: 'Netto').first
+  end
+
+  before(:all) do
+    TestData.create_netto
+  end
+
+  let(:translator) { Polo::SqlTranslator.new(netto, Polo::Configuration.new(adapter: :mysql)) }
+
+  describe '#ignore_transform' do
+    it do
+      insert_netto = [%q{INSERT IGNORE INTO "chefs" ("id", "name", "email") VALUES (1, 'Netto', 'nettofarah@gmail.com')}]
+
+      records = translator.records
+      inserts = translator.inserts
+      translated_sql = ignore_transform(inserts, records)
+      expect(translated_sql).to eq(insert_netto)
+    end
+  end
+
+
+  describe '#on_duplicate_key_update' do
+    it do
```

Comment:
`it 'appends ON DUPLICATE KEY UPDATE with all values to the current INSERT statement'`